qt/widgets/common/inc/MantidQtWidgets/Common/ManageUserDirectories.h

```diff
@@ -18,7 +18,8 @@ class EXPORT_OPT_MANTIDQT_COMMON ManageUserDirectories : public QDialog {
   Q_OBJECT
 
 public:
-  ManageUserDirectories(QWidget *parent = nullptr);
```

Comment:
Have these changes been put in this PR by accident? They don't seem relevant to the description.

Reply:
👍 	This is work from #24783, must have added the files at some point by accident